src/TextField/EnhancedTextarea.js

```diff
@@ -136,7 +135,9 @@ class EnhancedTextarea extends Component {
     const {prepareStyles} = this.context.muiTheme;
     const styles = getStyles(this.props, this.context, this.state);
     const rootStyles = Object.assign({}, styles.root, style);
-    const textareaStyles = Object.assign({}, styles.textarea, textareaStyle);
+    const textareaStyles = Object.assign({}, styles.textarea, textareaStyle, {
```

Comment:
I would switch the order of `textareaStyle` and `{height: this.state.height,}` as the customization property `textareaStyle` should probably always have a higher priority.